src/components/containers/EventCard/EventCard.tsx

```diff
@@ -47,6 +58,12 @@ export const EventCard: React.FC<EventCardProps> = ({
                 {startDate &&
                     <StartDateChip date={startDate}/>}
             </div>
+                {timeToEvent <= 72 && timeToEvent >= 0 ? 
+                <div className={styles.timeToEvent}>
+                    <span>{timeToEvent}h do rozpoczęcia</span>
+                </div> :
+                ''
+                }
```

Comment:
tu zamiast składni 
`bool ? <widok/> : ''`

można użyć 
`bool && <widok/>`

tak jak zrobiłeś to wyżej 😄